src/polyfills.js

```diff
@@ -48,7 +48,8 @@ installGetBoundingClientRect(self);
 // TODO(jridgewell): Ship custom-elements-v1. For now, we use this hack so it
 // is DCE'd from production builds. Note: When the hack is removed, remove the
 // @suppress {suspiciousCode} annotation at the top of this file.
-if ((false && isExperimentOn(self, 'custom-elements-v1')) || getMode().test) {
+if ((false && isExperimentOn(self, 'custom-elements-v1')) ||
+    (getMode().test && !getMode().testIe)) {
   installCustomElements(self);
```

Comment:
So the goal for CE v1 polyfill is to remove the DRE polyfill entirely. Is there a page that shows what's missing for CE v1 to support IE 11?

Reply:
I don't have a production page but this is what happens on Travis when you run integration tests in IE without the DRE Polyfill - [registerElement error](https://travis-ci.org/ampproject/amphtml/jobs/529879000#L1699-L1714)

Reply:
Mmm, it seems it's not finding `'customElements' in window`, so it's trying `document.registerElement`. Since IE doesn't have that, it fails. I'm fine approving this for now, but we should investigate why CE isn't being polyfilled.

Reply:
Sounds good, I added a TODO for follow up.